entities.py

```diff
@@ -163,7 +181,7 @@ def __init__(self, name, pos, behavior, char, color, game_map):
         self._gold = 100
         # technically all actors will have a backpack, but since it's a singleton
         # and only player has access to it, it doesn't matter.
-        self.backpack = Backpack()
+        self.backpack = Backpack(registry)
```

Comment:
I think this should be the other way around, we access the backpack through the registry